src/input_common/drivers/sdl_driver.cpp

```diff
@@ -405,8 +405,9 @@ SDLDriver::SDLDriver(const std::string& input_engine_) : InputEngine(input_engin
     // not a generic one
     SDL_SetHint("SDL_JOYSTICK_HIDAPI_JOY_CONS", "1");
 
-    // Turn off Pro controller home led
-    SDL_SetHint("SDL_JOYSTICK_HIDAPI_SWITCH_HOME_LED", "0");
+    // Disable hidapi driver for xbox. Already deafault on Windows, this causes conflict with native
+    // driver on Linux.
+    SDL_SetHint("SDL_HINT_JOYSTICK_HIDAPI_XBOX", "0");
```

Comment:
You probably meant this or "SDL_JOYSTICK_HIDAPI_XBOX". The other text equivalent does nothing
```suggestion
    SDL_SetHint(SDL_HINT_JOYSTICK_HIDAPI_XBOX, "0");
```

Reply:
Can confirm that the proposed change actually fixes the issue. The `xpad` driver no longer dies when launching yuzu with this applied.